test/integration/targets/yum_repository/tasks/main.yml

```diff
@@ -138,6 +140,7 @@
         file: "{{ yum_repository_test_repo.name ~ 2 }}"
         ip_resolve: 4
         keepalive: no
+        module_hotfixes: no
```

Comment:
Shouldn't we have at least test case with something besides the default value for the new parameter?

Reply:
Eventually, I was referring to keepalive option, which is added the same by default. Also, afaik, when keepalive or module_hotfixes are not set - they wouldn't be added to the repository itself. But when they explicitly set to false - they would be written into the repo file.

Reply:
I see how this is working now. The default is actually `None` per the argspec, which would be ignored when `allowed_params` is processed in `YumRepo.add()`. In that case, we should change the documentation section to reflect this and this test is ok.